tensorboard/backend/event_processing/io_wrapper.py

```diff
@@ -162,6 +167,59 @@ def ListRecursivelyViaGlobbing(top):
         current_glob_string = os.path.join(current_glob_string, "*")
         level += 1
 
+import queue
+import time
+import os
+import threading
+class listdir_bfs:
```

Comment:
nit: I think python convention should be camel case with inheritance to object
```
class ListDirBFS(object):
```

Reply:
changed to ListRecursivelyViaBFSWalking